File: E_Commerce_VS/Models/Database/Entidades/ProductoCarrito.cs
namespace E_Commerce_VS.Models.Database.Entidades
{
    public class ProductoCarrito
    {
        public int Id { get; set; }

        public long? ProductoId { get; set; }
        public int? CarritoId { get; set; }
        public int? Cantidad { get; set; }

        
        public Carrito Carrito { get; set; }
        public Producto Producto { get; set; }
    }
}

File: ECommerce_Angular/src/app/models/producto-carrito.ts
import { Product } from './product';

export class ProductoCarrito {
  productoId: number;
  carritoId: number;
  cantidad: number;
<<<<<<< HEAD
  producto: Product;
  id: any;
=======
  producto: Product; // Incluimos el producto completo
>>>>>>> origin/salperro2

  constructor(productoId: number, carritoId: number, cantidad: number, producto: Product) {
    this.productoId = productoId;
    this.carritoId = carritoId;
    this.cantidad = cantidad;
    this.producto = producto; // Asignamos el producto completo
  }
}
